controller/hero_chassis_controller:
  front_left_pid:
    p: 0.3
    i: 0.1
    d: 0.0
    i_clamp_min: -1.0
    i_clamp_max: 1.0
    antiwindup: true

  front_right_pid:
    p: 0.3
    i: 0.1
    d: 0.0
    i_clamp_min: -1.0
    i_clamp_max: 1.0
    antiwindup: true

  back_left_pid:
    p: 0.3
    i: 0.1
    d: 0.0
    i_clamp_min: -1.0
    i_clamp_max: 1.0
    antiwindup: true

  back_right_pid:
    p: 0.3
    i: 0.1
    d: 0.0
    i_clamp_min: -1.0
    i_clamp_max: 1.0
    antiwindup: true